README.md

```diff
@@ -47,4 +47,9 @@ apiName is one of:
 
  contractAddress in hexadecimal format (i.e 0x1F98431c8aD98523631AE4a59f267346ea31F984)
 
- outDir specifies optionally where the sources will be downloaded relative to current directory
\ No newline at end of file
+ outDir specifies optionally where the sources will be downloaded relative to current director
```

Comment:
Please fix the small typo introduced :)

Reply:
- Type fixed
- Also added new descriptive line L55 in readme.md